DrivewayDentDeletion/Operators/configure-postgres.sh

```diff
@@ -12,29 +12,60 @@ function usage {
   echo "Usage: $0 -n <namespace>"
 }
 
-namespace="cp4i"
+NAMESPACE="cp4i"
 
 while getopts "n:r:" opt; do
   case ${opt} in
-    n ) namespace="$OPTARG"
+    n ) NAMESPACE="$OPTARG"
       ;;
     \? ) usage; exit
       ;;
   esac
 done
 
-namespace=$(echo $namespace | sed 's/-/_/g')
-postgresPod=$(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_POD=$(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_SVC="$(oc get cm postgres-config -ojson | jq '.data["postgres.env"] | split("\n  ")' | grep DATABASE_SERVICE_NAME | cut -d "=" -f 2- | tr -dc '[a-z0-9-]\n').postgres.svc.cluster.local"
```

Comment:
oc get cm postgres-config -ojson  <-- missing a space here

Reply:
space between option and argument is optional, but i'll add a space for consistency

Reply:
yes please